aries/aries_vcx_core/src/wallet/indy/mod.rs

```diff
@@ -1,16 +1,46 @@
-use indy_api_types::domain::wallet::IndyRecord;
-use serde::{Deserialize, Serialize};
-use typed_builder::TypedBuilder;
+use async_trait::async_trait;
+use indy_api_types::domain::wallet::{default_key_derivation_method, IndyRecord};
+use vdrtools::{indy_wallet::iterator::WalletIterator, Locator};
 
 use self::indy_tags::IndyTags;
-use super::base_wallet::{record::Record, BaseWallet};
+use super::base_wallet::{
+    did_wallet::DidWallet,
+    issuer_config::IssuerConfig,
+    record::{AllRecords, PartialRecord, Record},
+    BaseWallet,
+};
 use crate::{errors::error::VcxCoreResult, WalletHandle};
 
 mod indy_did_wallet;
 mod indy_record_wallet;
-mod indy_tags;
-pub mod internal;
-pub mod wallet;
+pub(crate) mod indy_tags;
```

Comment:
Does not need to be made public (again).

Reply:
Changed.